src/js/track.js

```diff
@@ -44,7 +44,29 @@ Track.prototype.getSelectedSong = function() {
 };
 
 Track.prototype.normalizeSearchQuery = function(query) {
-	var normalized = query;
+	var removeableTags = ["official music video", "lyric video",  "out now",  "cover art",  "official video", "hd", "hq", "lyrics", "radio edit", "radio mix"];
```

Comment:
If you expand the check for these tags to be wrapped in parens/brackets as I requested in the other comment, you should also add these tags: `official audio`, `official song`, `audio`, `video`, `explicit version`, `clean version`,